docs/rule/no-dom-events.md

```diff
@@ -0,0 +1,27 @@
+## no-dom-events
+
+Mixing Ember actions and DOM events can lead to confusing and unexpected behavior. For a comprehensive explanation of why, read [Deep Dive on Ember Events]. As a result, it is recommended to stick entirely to Ember actions.
+
+This rule **forbids** the following:
+
+```hbs
+<button onclick={{action (action "submit")}}>Submit</button>
```

Comment:
The nested `(action` is not needed here (AFAICT it doesn't do anything...).

I'd write this as:

```hbs
<button onclick={{action "submit"}}>Submit</button>
```

And the allowed version would be:

```hbs
<button {{action "submit"}}>Submit</button>
```

Reply:
Fixed.